vscode-lean4/package.json

```
@@ -64,6 +64,21 @@
 					"default": true,
 					"markdownDescription": "Enable eager replacement of abbreviations that uniquely identify a symbol."
 				},
+				"lean4.editor.occurrenceHighlightingMode": {
+					"type": "string",
+					"description": "Highlight occurrences using Lean's semantic information, by string matching, or both",
```

```suggestion
					"description": "Highlight occurrences of a symbol using Lean's semantic information, by string matching, or both",
```
Maybe slightly clearer as to what it is that's occurring?

---

I intentionally avoided this phrasing because the string-based highlighting doesn't match symbols (which is why I want to turn it off - I find it undermines the usefulness of the highlights if I can't trust them). But perhaps I'm being too pedantic here.

How do you feel about the phrasing in #377?

---

Gotcha! The other PR is worded well.

---

Thanks! I tried hard to thread that needle, glad it succeeded